e2e/tests/utils/puppeteer.ts

```diff
@@ -7,14 +8,24 @@ export { Browser } from "puppeteer";
 export async function getPage(options: { browser: Browser }): Promise<Page> {
   const page = await options.browser.newPage();
 
-  await page.goto(`http://localhost:${BACKEND_PORT}`);
+  await page.evaluateOnNewDocument(
+    (theme) => {
+      // @ts-ignore
+      localStorage.setItem("theme", theme);
+    },
+    // TODO - test dark mode https://github.com/rocket-connect/graphql-debugger/issues/151
+    JSON.stringify({ state: { theme: "light" }, version: 0 }),
```

Comment:
@kreshnnikgashi here we hardcode local storage to always run the e2e in light mode, this way all the color assertions will not be flaky on each machine. 

I made an issue here to track the work of adding dark mode into the e2e testing suite: https://github.com/rocket-connect/graphql-debugger/issues/151